.github/workflows/go.yaml

```diff
@@ -16,7 +16,10 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v3
       with:
-        go-version: 1.19
+        go-version: 1.21
+
+    - name: Get sets
+      run: go get k8s.io/apimachinery/pkg/util/sets
```

Comment:
why do we need this?

Reply:
https://github.com/openshift/installer/blob/master/pkg/types/powervs/powervs_regions.go#L132-L135

Reply:
you just need to run `go mod tidy` and add any autogenerated go.sum file, lets not add this while building

Reply:
Changed, thanks!